src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/noLineWrap/InputNoLineWrapBad.java

```diff
@@ -0,0 +1,28 @@
+package com.puppycrawl.tools.
```

Comment:
@divsidhu-26 , please use "git mv old-path-name new-path-name" instead of coping file manually.